### PR TITLE
add ingress for schema registry

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Comment by jhoblitt** [comment]
Currently based on #29

File: kafka.tf [hunk 1]
```diff
@@ -1,3 +1,7 @@
+locals {
+  registry_fqdn = "${local.dns_prefix}registyb-${var.deploy_name}.${var.domain_name}"
```
Comment (afausti) [inline kafka.tf]: fix typo in "registry"

**Review by afausti — APPROVED** [review]
LGTM with the caveat that if we manage to run the SAL Kafka producers inside the cluster this won't be necessary. Also,  if in the end we expose the schema registry to outside the cluster we'll need some sort of authentication. I think Confluent Platform 5.3 Helm charts may help on that.

File: kafka.tf (hunk 1 above)
Comment (jhoblitt) [reply kafka.tf]: fixed

**Comment by jhoblitt** [comment]
rebased on master instead of #29 

re auth: nginx-ingress does support basic http auth: https://kubernetes.github.io/ingress-nginx/examples/auth/basic/

**Pull request merged**